libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -1,42 +1,48 @@
-use std::cmp::{max, min};
-use std::str::FromStr;
-use std::sync::Arc;
-use std::time::{SystemTime, UNIX_EPOCH};
+use crate::invoice::parse_invoice;
+use crate::models::{
+    Config, GreenlightCredentials, LnPaymentDetails, Network, NodeAPI, NodeState, PaymentDetails,
+    PaymentType, SyncResponse, UnspentTransactionOutput,
+};
+use crate::persist::db::SqliteStorage;
+use crate::{Channel, ChannelState, NodeConfig};
+
+use lightning::util::message_signing;
+#[allow(unused)]
+use zbase32;
 
 use anyhow::{anyhow, Result};
 use bitcoin::bech32::{u5, ToBase32};
 use bitcoin::secp256k1::ecdsa::{RecoverableSignature, RecoveryId};
-use bitcoin::secp256k1::Secp256k1;
-use bitcoin::util::bip32::{ChildNumber, ExtendedPrivKey};
+use bitcoin::secp256k1::PublicKey;
 use ecies::utils::{aes_decrypt, aes_encrypt};
 use gl_client::node::ClnClient;
 use gl_client::pb::amount::Unit;
+
 use gl_client::pb::cln::{
     self, CloseRequest, ListclosedchannelsClosedchannels, ListclosedchannelsRequest,
     ListpeerchannelsRequest,
 };
 use gl_client::pb::{
-    Amount, Invoice, InvoiceRequest, InvoiceStatus, OffChainPayment, PayStatus, Peer,
-    WithdrawResponse,
+    Amount, Invoice, InvoiceRequest, InvoiceStatus, OffChainPayment, PayStatus, WithdrawResponse,
 };
 use gl_client::scheduler::Scheduler;
 use gl_client::signer::Signer;
 use gl_client::tls::TlsConfig;
 use gl_client::{node, pb, utils};
+
+use bitcoin::secp256k1::Secp256k1;
+use bitcoin::util::bip32::{ChildNumber, ExtendedPrivKey};
+use gl_client::pb::Peer;
 use lightning_invoice::{RawInvoice, SignedRawInvoice};
 use serde::{Deserialize, Serialize};
+use std::cmp::{max, min};
+use std::str::FromStr;
+use std::sync::Arc;
+use std::time::{SystemTime, UNIX_EPOCH};
 use strum_macros::{Display, EnumString};
 use tokio::sync::{mpsc, Mutex};
 use tonic::Streaming;
 
-use crate::invoice::parse_invoice;
-use crate::models::{
-    Config, GreenlightCredentials, LnPaymentDetails, Network, NodeAPI, NodeState, PaymentDetails,
-    PaymentType, SyncResponse, UnspentTransactionOutput,
-};
-use crate::persist::db::SqliteStorage;
-use crate::{Channel, ChannelState, NodeConfig};
-
```

Comment:
I think only `use bitcoin::secp256k1::PublicKey;` and `use lightning::util::message_signing;` are necessary here. The other changes can be dropped. Also the zbase32 is not necessary, because it's referenced directly in the file.